frescobaldi.yaml

```diff
@@ -0,0 +1,107 @@
+---
+name: frescobaldi
+buildsystem: simple
+build-commands:
+  - make -C linux
+  - python3 setup.py install --prefix=/app --root=/
+sources:
+  - type: archive
+    url: https://github.com/frescobaldi/frescobaldi/releases/download/v3.1.2/frescobaldi-3.1.2.tar.gz
+    sha256: 5c2cffb8282cd9faef1585808bd800d1eb3c0db4cc464a61ce8576dbf7ef9b20
+  - type: patch
+    path: patches/frescobaldi-fix-invalid-metainfo-file.patch # can be removed in next release
+
+modules:
+  # OpenJPEG is needed to build Poppler
+  - shared-modules/openjpeg/openjpeg.json
+
+  - name: poppler
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_INSTALL_LIBDIR=/app/lib
+      - -DCMAKE_INSTALL_INCLUDEDIR=/app/include
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: https://poppler.freedesktop.org/poppler-0.80.0.tar.xz
+        sha256: 4d3ca6b79bc13b8e24092e34f83ef5f387f3bb0bbd7359a6c078e09c696d104f
+
+  - name: sip
+    buildsystem: simple
+    build-commands:
+      - python3 configure.py
+        --bindir=/app/bin
+        --destdir=/app/lib/python3.7/site-packages
+        --incdir=/app/include
+        --sipdir=/app/share/sip
+        --stubsdir=/app/lib/python3.7/site-packages
+        --sip-module=PyQt5.sip
+      - make -j $FLATPAK_BUILDER_N_JOBS
+      - make install
+    sources:
+      - type: archive
+        url: https://www.riverbankcomputing.com/static/Downloads/sip/4.19.19/sip-4.19.19.tar.gz
+        sha256: 5436b61a78f48c7e8078e93a6b59453ad33780f80c644e5f3af39f94be1ede44
+    cleanup:
+      - /bin
+
+  - name: pyqt5
+    config-opts:
+      - --disable-static
+      - --enable-x11
+    sources:
+      - type: archive
+        url: https://www.riverbankcomputing.com/static/Downloads/PyQt5/5.13.1/PyQt5_gpl-5.13.1.tar.gz
+        sha256: 54b7f456341b89eeb3930e786837762ea67f235e886512496c4152ebe106d4af
```

Comment:
As for PyQt5, I use the version on pypi, but being a minor revision up, it might need to be verified working:
```
        {
          "type" : "archive",
          "url": "https://files.pythonhosted.org/packages/4d/81/b9a66a28fb9a7bbeb60e266f06ebc4703e7e42b99e3609bf1b58ddd232b9/PyQt5-5.14.2.tar.gz",
          "sha256": "bd230c6fd699eabf1ceb51e13a8b79b74c00a80272c622427b80141a22269eb0"
        },
```

Reply:
you can use that mirror otherwise https://distfiles.macports.org/py-pyqt5/

Reply:
I can build SIP 4.19.24, but PyQt5 5.14.2 is failing with the [same error discussed here](https://github.com/meta-qt5/meta-qt5/pull/334). Upgrading PyQt5 to 5.15.0 fixed the problem:

```
https://files.pythonhosted.org/packages/8c/90/82c62bbbadcca98e8c6fa84f1a638de1ed1c89e85368241e9cc43fcbc320/PyQt5-5.15.0.tar.gz

c6f75488ffd5365a65893bc64ea82a6957db126fbfe33654bcd43ae1c30c52f9
```

I'll have to verify that everything works fine.

Reply:
I build it fine with either 5.14 or 5.15 in the LMMS flatpak.

https://github.com/flathub/io.lmms.LMMS/blob/master/io.lmms.LMMS.json#L108

Reply:
Downgrading SIP to 4.19.22 seems necessary to build PyQt 5.14.2.